tests/input/BINARY_OP.py

```diff
@@ -0,0 +1,4 @@
+def add(a, b):
+    return a + b
+
+add(1, 3)
```

Comment:
Probably doesn't need to include a function to generate the opcode(s) -- might as well get rid of the function to reduce what's being tested.  Also, please ensure files end with a newline.